### ISSUE TITLE
Attester slashing 21759

### ISSUE THREAD

**Issue by dankrad** [goerli/medalla#40]
https://medalla.beaconcha.in/validator/21759
Eth1 address: 0x694e5550d604F176B037Afd776722a79D3616d6d
Graffiti (on other validator by same Eth1 address): poapaU5VUNYE8XawN6/XdnIqedNhbW0A

**Comment by rileyjt** [comment]
My setup that caused this: 

- Generated 8 validator keys through the launch pad. 
- I got enough GoEth to make 5 deposits.
- Setup Prysm and imported validator keys for genesis block. (my confusion was here, It said 5 validators were active and waiting on the genesis block, but it actually imported all 8 keys and the 3 without deposits were silently "dormant")
- Got more GoEth and made 3 deposits for second set of keys
- Setup Lighthouse and imported the second set of keys
- Genesis happened - the first set of 5 Prysm validators all functioned normally.
- Waited two days for the second set of deposits to clear and the second set of validators to get through the queue
- The second set of validators activated on both Prysm and Lighthouse where one was slashed a few hours later

I believe I have this resolved. There was no easy way to remove the validator keys from the Prysm wallet, but I deleted the wallet and re-imported validator keys with only the first set of 5. The second set of 3 (now 2) are continuing to run on Lighthouse.
